engine/routes.py

```diff
@@ -16,6 +16,9 @@
 from util.releasenotes import ReleaseNotes
 from util.timeout import set_timeout, set_inactivity_timeout
 
+LOCAL_ASYNC_DIR = app.config['LOCAL_ASYNC_DIR']
+FINAL_ASYNC_DIR = app.config['FINAL_ASYNC_DIR']
```

Comment:
Can you add a change to config/default.py so I don't need to continue "fixing" it in the releases? Since we migrated from uframe-test to ooiufs11 last August I've had to add the following to config/local.py to override the config/default.py settings:
LOCAL_ASYNC_DIR = '/home/local1/async_results'
FINAL_ASYNC_DIR = '/home/local2/async_results'

Reply:
I think this needs some discussion. It strikes me as exactly the sort of thing that is going to vary from environment to environment. We have issues because we source control one config file, but we use it in production, locally, and on test servers, each of which could need different configurations. I don't have any problem making the particular change you ask for, but I know for example, that I would have to either change it on my local machine or change my directory structure. It seems like part of a bigger issue that needs attention.

Reply:
Per discussion this morning, this change will be handled by another ticket and there will be a general config file discussion.